Enable half type support for unique cpu

zyan0 opened the pull request:
Test Plan: CI

Differential Revision: D42326527



cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

jgong5 reviewed (approved):
Is this covered by any UT?

Commit:
Enable half type support for unique cpu (#91666)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/91666

Test Plan: CI

Reviewed By: bixue2010

Differential Revision: D42326527

fbshipit-source-id: 76c4a7b772cd18642296a07c91fbfa200a1e608e

zyan0 commented:
@pytorchbot merge

zyan0 commented:
@pytorchbot merge
